### ISSUE TITLE
Calculate property 2 times not working as expected 

### ISSUE THREAD

**Issue by RobertoAbraul** [cerberustesting/cerberus-source#1101]
Hi,

after calculate the same property a second time cerberus uses the first one that were calculated.

We have the following behavior:
Step1:
1. calculate property "x" = OK
2. use calculated property "x" to make an action = OK
 
step2:
1. calculate property "x" = OK
2. use calculated property "x" to make an action = KO

On step 2.2 cerberus uses value calculated on Step1.1.

Is this the expected behavior?

http://cerberus.siege.red/Cerberus/ExecutionDetail.jsp?id_tc=41006636

![step1](https://cloud.githubusercontent.com/assets/14328346/21720828/3da6a1d2-d41d-11e6-8564-e0cd490bf5ed.png)
![step2](https://cloud.githubusercontent.com/assets/14328346/21720858/6176cfba-d41d-11e6-8d84-ac2ccec634e1.png)

Can you please take a look?

Thanks

**Comment by vertigo17** [comment]
In your example, the property that fail is %SIZE_DROPDOWN%.
In fact, that property has no calculateProperty action and as a consequence is not calculated but only reused from the past calculation (from the 1st step).
Change the calculateProperty action from SIZE_DROPDOWN_ARTICLE to SIZE_DROPDOWN and it will force calculation of both.

The global rule is that whenever you use a property, it gets calculated automatically the 1st time (no need to specify a calculateProperty action). Anytime you want to force a new calculation on a specific timing, you need the calculateProperty action that will calculate the given property but also any other property used in it.
In other words, calculateProperty actions should almost never be used unless you really need to control the timing of the calculation of given property.

Please close the ticket in order to confirm the explanation is clear.
